### PR TITLE
Updated plots and -2 values CSV & ipynb for Geetha

### PR DESCRIPTION
**Pull request by zacharymeurer**
Updated plots since week 2 demo review (adding vertical lines to graphs to indicate solstices & plotting multiple lines on the same graph). 

Wrote python notebook to create a CSV file, adding new columns to alldata.csv. The new columns indicate whether a phenophase should be reported as -2 (e.g. The open fruit phenophase **does not** appear in mangos, but citizens report values other than -2 in the open fruit column) or is mistakenly reported as -2 (e.g. The mature leaves phenophase **does** appear in mangos, but citizens report values of -2 in the mature leaves column) for each phenophase. I do this for the top 10 most prevalent species in the alldata.csv file. 

The possible values in the new columns are 0, 1, & 2.
- 0 : Valid
- 1 : Mistakenly reported as -2 (false positive)
- 2 : Mistakenly reported as not -2.

The notebook and CSV files are in this branch. Once this pull request is approved Ana can send the CSV file over to Geetha with the .

### CODE REVIEW

**Review by jacobe90 — APPROVED** [review]
Replaced citizen_data directory contents with citizen data from dev. Looked at `-2_values.ipynb` and `plotting.ipynb`, both look great! I particularly like the way of adding a column to label points as either being falsely -2, falsely non -2, or correct.

Approved!

File: pvt_cleaning.ipynb [hunk 1]
```diff
@@ -0,0 +1,1582 @@
+{
```
Comment (colettebas) [inline pvt_cleaning.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #28.        # - get rid of null values
</pre><p>Make sure there is a task for this in notion!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (colettebas) [inline pvt_cleaning.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #71.                if not isinstance(cat_vector_for_week, str):
</pre><p>Is this needed to fill in data for the years before a category started being records?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (colettebas) [inline pvt_cleaning.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    [1, 2, 3].remove(4)
</pre><p>I think this needs to be removed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

File: pvt_validation.ipynb [hunk 2]
```diff
@@ -0,0 +1,197 @@
+{
```
Comment (colettebas) [inline pvt_validation.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    # weird, pvt_andhra_pradesh has unmatching duplicates
</pre><p>Was this the only place where this happened? I wonder if it was an issue with types. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

**Comment by zacharymeurer** [comment]
> View / edit / reply to [this conversation](https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/6/discussion/#comment-2142733824) on ReviewNB
> 
> _colettebas commented on 2024-05-31T17:58:11Z_ ----------------------------------------------------------------
> 
> Line #10.        plt.show()
> The y axis needs a label here. Also, I would start to think about a more readable label for the x axis since anyone who is not on your team will not know what -2, -1, etc. mean.

Good idea about readability. If this plot goes into the final report that will definitely be something we need to consider. For now as a quick fix I'll print a key alongside the graph.

**Review by AnaSof0 — APPROVED** [review]

**Pull request merged**